README.md

```diff
@@ -1,83 +1,26 @@
 Takeaway Challenge
-==================
-```
-                            _________
-              r==           |       |
-           _  //            |  M.A. |   ))))
-          |_)//(''''':      |       |
-            //  \_____:_____.-------D     )))))
-           //   | ===  |   /        \
-       .:'//.   \ \=|   \ /  .:'':./    )))))
-      :' // ':   \ \ ''..'--:'-.. ':
-      '. '' .'    \:.....:--'.-'' .'
-       ':..:'                ':..:'
-
- ```
-
-Instructions
 -------
+<br/>
+The goal was to create a platform that would allow a user to place a takeway order. I decided to break down the project into four classes. The `Dish` and `Menu` classes were created to provide future expansion. I could have set a menu in the initialize method of the `Takeaway` class but I thought this would be cleaner and more future proof. 
 
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
+The `Takeaway` class provides the functionality of the program:
 
-Task
------
+- `list_dishes` to get a list of all available dishes
+- `order` lets you add dishes to your order
+- `check_order` shows a breakdown of the order with prices
+- `complete_order` sends out a confirmation text with expected delivery time
 
-* Fork this repo
-* Run the command 'bundle' in the project directory to ensure you have all the gems
-* Write a Takeaway program with the following user stories:
+This class is injected with `Menu.new` by default but could be provided with another menu class if this were required. It is also injected with `SMS.new`, which uses TWILIO API to send out a confirmation text for the order.
 
-```
-As a customer
-So that I can check if I want to order something
-I would like to see a list of dishes with prices
+In order to protect senstitive information storing API authentication data and phone numbers, these were all placed as `ENV` variables inside a .env file which is not added to version control.
 
-As a customer
-So that I can order the meal I want
-I would like to be able to select some number of several available dishes
+<br/>
 
-As a customer
-So that I can verify that my order is correct
-I would like to check that the total I have been given matches the sum of the various dishes in my order
-
-As a customer
-So that I am reassured that my order will be delivered on time
-I would like to receive a text such as "Thank you! Your order was placed and will be delivered before 18:52" after I have ordered
+Instructions
```

Comment:
I need to add this, you reminded me.

So, as you can probably tell: the README is really good!